app/assets/stylesheets/groups.scss

```diff
@@ -30,7 +30,7 @@
 }
 
 .display_pane {
-  border: 1px solid $light-grey;
+  border: 1px solid $background-main;
```

Comment:
Similar to a previous comment, doesn't this cause the border to blend into the background? (I won't comment at other places I see this.)

Reply:
Definitely a lot of places with this, hopefully I'm catching them all!

Reply:
**Fixed**:
Line 126 of exam_templates.scss is caught and fixed.
Line 140 of exam_templates.scss is caught and fixed.

**Questions**:
Line 133 of SyntaxHighlighter.scss has a similar occurrence but I am confused about where that applies. I left a comment on that file.

Lines 33 and 46 in groups.scss are essentially the same as the ones in exam_templates.scss, yet I cannot find where they are used for graders in either assignment settings or the graders list in users. All of the lines were authored in the same commit by Han in revision 1736b25. I think I should just removed these classes in the groups.scss file, since they are defined in the exam_templates.scss anyway. Can I do so?

**No fix needed**:
Line 795 of _markus.scss has a similar occurrence which is intentional and correct.
Line 462 of _navigation.scss has a similar occurrence which is intentional and correct.